accounts.go

```diff
@@ -351,7 +351,7 @@ func (ah *AccountHandler) newPassword(w http.ResponseWriter, r *http.Request) {
 
 		password, repeatPassword, currentPassword := r.FormValue("Password"), r.FormValue("RepeatPassword"), r.FormValue("CurrentPassword")
 
-		if !account.NeedsPasswordReset() {
+		if !account.NeedsPasswordReset() && (config.Accounts.AdminPasswordOverride == "" && account.Name == adminUserName) {
```

Comment:
if the user account is not admin, this will never be true, and the current password check won't occur. so you'd be able to change the account password without knowing the current one